README.md

```diff
@@ -14,24 +14,29 @@ The committee consists of 6 members:
 * Emily Pillmore @emilypi (term ending January 2023)
 * Melanie Brown @cigsender (term ending January 2025)
 * Andrew Lelechenko @Bodigrim (chair, term ending January 2025)
+* Tom Ellis @tomjaguarpaw (term ending February 2025)
 
 The committee makes decisions by simple majority voting.
 
 ## Contacts
 
-Members of the public are suggested to contact CLC primarily via this issue tracker.
+Members of the public are suggested to contact CLC primarily via [this issue tracker](https://github.com/haskell/core-libraries-committee/issues).
 As a secondary medium with more limited visibility one can use a mail list
 core-libraries-committee at haskell dot org
 ([public archives](https://groups.google.com/g/haskell-core-libraries)).
```

Comment:
I would really rather avoid section numbering, because it is a headache to maintain in markdown. I think it's not that long to require cross-references so far, maybe in the next iteration.